### PR TITLE
Tpwdcee 6288 a51try configuration

### PR DESCRIPTION
**Pull request by tatsta**

### CODE REVIEW

File: .bin/run-ui-tests.sh [hunk 1]
```diff
@@ -1,67 +1,68 @@
 #!/bin/bash
-# Shop System SDK:
+# Shop System Plugins:
```
Comment (brkicadis) [inline .bin/run-ui-tests.sh:2]: This should be Shop System Extensions:

- https://jira.wirecard.sys/browse/TPWDCEE-4224 here is stated.

File: .bin/run-ui-tests.sh [hunk 2]
```diff
@@ -1,67 +1,68 @@
...
+  TEST_GROUP="${MAJOR_RELEASE}"
 fi
 
-# run UI tests
-vendor/bin/codecept run acceptance -g ${TEST_GROUP} --html --xml
+#setup codeception and dependencies
+rm -rf composer.lock
+
+#get shopsystem-ui-testsuite project
+git clone  --branch TPWDCEE-6288-a51try-configuration https://github.com/wirecard/shopsystems-ui-testsuite.git
```
Comment (brkicadis) [inline .bin/run-ui-tests.sh:48]: Change branch name to master. And also, why git clone and not composer require? Is there a reason?

File: .bin/run-ui-tests.sh [hunk 3]
```diff
@@ -1,67 +1,68 @@
...
+cd shopsystems-ui-testsuite
```
Comment (brkicadis) [inline .bin/run-ui-tests.sh:49]: If you change it to composer require, then here "shopsystem-ui-testsuite".

File: .bin/run-ui-tests.sh [hunk 4]
```diff
@@ -1,67 +1,68 @@
...
+echo "Installing shopsystems-ui-testsuite dependencies"
+docker run --rm -it --volume $(pwd):/app prooph/composer:7.2 install --dev
+
+export SHOP_SYSTEM="${SHOP_SYSTEM}"
+export SHOP_URL="${NGROK_URL}"
+export EXTENSION_VERSION="${GIT_BRANCH}"
+export DB_HOST="${MAGENTO2_DB_HOST%%:*}"
+export DB_NAME="${MAGENTO2_DB_NAME}"
+export DB_USER="${MAGENTO2_DB_USER}"
+export DB_PORT="${MAGENTO2_DB_HOST#*:}"
```
Comment (brkicadis) [inline .bin/run-ui-tests.sh:59]: Why don't we use separate host and port?

File: .bin/start-shopsystem.sh [hunk 5]
```diff
@@ -1,55 +1,72 @@
 #!/bin/bash
-# Shop System SDK:
+# Shop System Plugins:
```
Comment (brkicadis) [inline .bin/start-shopsystem.sh:2]: Same here.

File: .bin/run-ui-tests.sh (hunk 2 above)
Comment (tatsta) [reply .bin/run-ui-tests.sh:48]: I will change to master before merging, thanks for reminder. 
I tried to do it through composer, but the shopsystems-ui-testsuite dependencies are not compatible with magento2 dependencies, so everything breaks.

File: composer.json [hunk 6]
```diff
@@ -1,52 +1,52 @@
...
+        ]
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "WirecardTest\\ElasticEngine\\": "Test"
+        }
     }
-  }
-}
+}
```
Comment (brkicadis) [inline composer.json:52]: Empty space.

File: .bin/start-shopsystem.sh [hunk 7]
```diff
@@ -1,55 +1,72 @@
...
+sleep 5
+#change hostname
+docker exec -ti ${MAGENTO2_CONTAINER_NAME} /opt/wirecard/apps/magento2/bin/hostname-changed.xsh "${NGROK_URL#*//}"
+
+#set cron to every minute
+docker exec -ti ${MAGENTO2_CONTAINER_NAME} /bin/sh -c "sed 's/15/1/g' /srv/http/vendor/wirecard/magento2-ee/etc/crontab.xml > /srv/http/vendor/wirecard/magento2-ee/etc/crontab1.xml"
+docker exec -ti ${MAGENTO2_CONTAINER_NAME} /bin/sh -c "cp /srv/http/vendor/wirecard/magento2-ee/etc/crontab1.xml /srv/http/vendor/wirecard/magento2-ee/etc/crontab.xml"
+
+# randomize PayPal orderNumber
+docker exec -ti ${MAGENTO2_CONTAINER_NAME} bash -c "sed -i 's/ = \$this->orderNumber\;/ = \$this->orderNumber . md5(time())\;/' /srv/http/vendor/wirecard/payment-sdk-php/src/Transaction/PayPalTransaction.php"
```
Comment (brkicadis) [inline .bin/start-shopsystem.sh:68]: nice :)

Comment (tatsta) [reply .bin/start-shopsystem.sh:68]: I made a follow up US to make it clean later on

**Pull request merged**